Doc/library/tarfile.rst

```diff
@@ -784,7 +784,7 @@ How to read a gzip compressed tar archive and display some member information::
    import tarfile
    tar = tarfile.open("sample.tar.gz", "r:gz")
    for tarinfo in tar:
-       print(tarinfo.name, "is", tarinfo.size, "bytes in size and is", end="")
+       print(tarinfo.name, "is", tarinfo.size, "bytes in size and is ", end="")
```

Comment:
Maybe this statement could use an f-string?

Reply:
Ack. Still this functional improvement is valuable.